integration-tests/contract-deploy-having-ticket.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { CONFIGS } from "./config";
+import { ticketCode, ticketStorage } from './data/code_with_ticket';
+import { ticketCodeProto14, ticketStorageProto14 } from './data/code_with_ticket_proto14';
+import { Protocols } from "@taquito/taquito";
+
+CONFIGS().forEach(({ lib, rpc, setup, protocol }) => {
+  const Tezos = lib;
+  const kathmandunet = protocol === Protocols.PtKathman ? test: test.skip;
+  const limanetAndAlpha = protocol === Protocols.PtLimaPtL || protocol === Protocols.ProtoALpha ? test: test.skip;
+
+  describe(`Test origination of a token contract using: ${rpc}`, () => {
+
+    beforeEach(async (done) => {
+      await setup();
+      done()
+    })
```

Comment:
semi colons 😄

Reply:
Quick note for next time, those should be added automatically if using `npm run commit`